docs/resources/access_package_assignment_policy.md

```diff
@@ -0,0 +1,218 @@
+---
+subcategory: "Identity Governance"
+---
+
+# Resource: azuread_access_package_assignment_policy
+This resources manages the policies within an access package.
+
+## API Permissions
+The following API permissions are required in order to use this resource.
+
+When authenticated with a service principal, this resource requires the following application role: `EntitlementManagement.ReadWrite.All`.
+
+When authenticated with a user principal, this resource requires `Global Administrator` directory role, or one of the `Catalog Owner` and `Access Package Manager` role in Idneity Governance.
+
+## Example Usage
+
+```terraform
+provider "azuread" {}
+
+resource "azuread_group" "test" {
+  display_name     = "test-group-%[1]d"
```

Comment:
```suggestion
  display_name     = "test-group"
```
The placeholders in the examples look slightly confusing. Other examples don't include them, so I would suggest to remove them.

Reply:
I'll leave this one as it is for now, the examples are copied from the code, so I'd like to keep them same.